app/services/katello/organization_creator.rb

```diff
@@ -86,8 +92,8 @@ def create_content_view_environment
 
     def create_library_environment
       @library_environment = Katello::KTEnvironment.where(
-        :name => "Library",
-        :label => "Library",
+        :name => DEFAULT_LIFECYCLE_ENV_NAME,
+        :label => DEFAULT_CONTENT_VIEW_LABEL,
```

Comment:
i think this should be DEFAULT_LIFECYCLE_ENV_NAME

which is probably the cause of the test failures

Reply:
errr  DEFAULT_LIFECYCLE_ENV_LABEL